summer-of-code/week-11/day-2.md

```diff
@@ -12,7 +12,19 @@ Today we will learn:
 * How to connect scenes
 
 
-### Trigger setup
+### Recognizing Tap gestures
+The following method will allow you to interact with any gameobject as long as it has a callider set as Trigger.
+
+```csharp
+void OnMouseUpAsButton()
+	{
+		//Do something
+		}
+
+```
+
+
+### Trigger - Trigger interaction setup
 
 When working with 2 or more targets, the sizes are important
 A pairing of 2 objects, one has to be the trigger and have a rigidbody inside (Kinematic).
```

Comment:
is `rigidbody` written like this or like `rigid body`

Reply:
when writing the code and looking inside the documentation it always appears as "rigidbody", I put the word in italic characters so that its clear that it is because of the context